sources/packages/web/src/views/aest/StudentAppealRequestsApproval.vue

```diff
@@ -0,0 +1,152 @@
+              >Cancel</v-btn
+            >
+            <v-btn color="primary" class="ml-2" @click="submit"
+              >Complete student request</v-btn
+            ></v-row
+          >
+        </template>
+      </appeal-requests-approval-form>
+    </full-page-container>
+  </v-container>
+</template>
+<script lang="ts">
+import { ref, onMounted, computed } from "vue";
+import FullPageContainer from "@/components/layouts/FullPageContainer.vue";
+import HeaderNavigator from "@/components/generic/HeaderNavigator.vue";
+import BodyHeader from "@/components/generic/BodyHeader.vue";
+import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
+import { useRouter } from "vue-router";
+import { StudentAppealService } from "@/services/StudentAppealService";
+import { useFormatters, useToastMessage } from "@/composables";
+import {
+  StudentAppealRequest,
+  StudentAppealApproval,
+  StudentAppealStatus,
+} from "@/types";
+import AppealRequestsApprovalForm from "@/components/aest/AppealRequestsApprovalForm.vue";
+import StatusChipRequestedAssessment from "@/components/generic/StatusChipRequestedAssessment.vue";
+
+export default {
+  components: {
+    HeaderNavigator,
+    FullPageContainer,
+    BodyHeader,
+    AppealRequestsApprovalForm,
+    StatusChipRequestedAssessment,
+  },
+  props: {
+    studentId: {
+      type: Number,
+      required: true,
+    },
+    applicationId: {
+      type: Number,
+      required: true,
+    },
+    appealId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props: any) {
+    const router = useRouter();
+    const toast = useToastMessage();
+    const { dateOnlyLongString } = useFormatters();
+    const studentAppealRequests = ref([] as StudentAppealRequest[]);
+    const appealStatus = ref(StudentAppealStatus.Pending);
+    const readOnly = computed(
+      () => appealStatus.value !== StudentAppealStatus.Pending,
+    );
+
+    onMounted(async () => {
+      const appeal =
+        await StudentAppealService.shared.getStudentAppealWithRequests(
+          props.appealId,
+        );
+      studentAppealRequests.value = appeal.appealRequests.map((request) => ({
+        id: request.id,
+        data: request.submittedData,
+        formName: request.submittedFormName,
+        approval: {
+          id: request.id,
+          appealStatus: request.appealStatus,
+          assessedDate: dateOnlyLongString(request.assessedDate),
+          assessedByUserName: request.assessedByUserName,
+          noteDescription: request.noteDescription ?? "",
+          showAudit: false,
+        },
+      }));
+      appealStatus.value = appeal.status;
+    });
+
+    const assessmentsSummaryRoute = {
+      name: AESTRoutesConst.ASSESSMENTS_SUMMARY,
+      params: {
+        applicationId: props.applicationId,
+        studentId: props.studentId,
+      },
+    };
+
+    const gotToAssessmentsSummary = () => {
+      router.push(assessmentsSummaryRoute);
```

Comment:
mm, not a blocker, just wanted to know, why `assessmentsSummaryRoute` is defined separately

Reply:
It is used in the header component
![image](https://user-images.githubusercontent.com/61259237/162039698-fc730e71-1bf0-46bd-bd9d-ae19b134d7a6.png)

Back button
![image](https://user-images.githubusercontent.com/61259237/162039761-d5ada3ca-288e-4e96-b54d-6e66e4e9f0f2.png)

On success
![image](https://user-images.githubusercontent.com/61259237/162039821-2616401f-4c33-40bb-a101-1a9fbc4cdb8c.png)

Header and others uses different types.

Reply:
mm..okay